Add GitHub issue forms (#5776)
redwoodjs/redwood: CONTRIBUTING.md

_Before interacting with the Redwood community, please read and understand our [Code of Conduct](https://github.com/redwoodjs/redwood/blob/main/CODE_OF_CONDUCT.md)._

> **Table of Contents**

Reviewer: @jtoar removed this because Github now gives you a ToC anyway

![image](https://user-images.githubusercontent.com/32992335/130697522-313317f8-21e5-4f71-8b8e-9690dbad412a.png)

> ## Creating a reproduction to include with issues
> Including a reproduction, either as a series of steps, a public Github repo with the problem you are facing or a Gitpod snapshot will definitely let us help you faster!

Author: ```suggestion
## Creating a Reproduction to Include with Issues
Are you about to open an issue? Including a reproduction, either as a series of steps, as a public GitHub repo, or as a Gitpod snapshot, will definitely let us help you faster!
```
